### ISSUE TITLE
ELB / route53 - InvalidChangeBatch: Tried to create an alias that targets XXXX, type A in zone YYYYY, but that target was not found.

### ISSUE THREAD

**Issue by quietlyconfident** [boto/boto#2519]
I'm using boto to create an ELB, then add an alias record of it to my route53 hosted zone.  When I attempt to do so, I sometimes get the error message: "route53 - InvalidChangeBatch: Tried to create an alias that targets XXXX, type A in zone YYYYY, but that target was not found."

I say sometimes because if I wait 3-5 minutes, then I can create the record without problem.  I suspect that this is because route53 will not create the record until the ELB instance is active.

However, the ELB object does not provide any documented way to determine whether it is up or not.  So there is not any way to know how long I need to wait to update the DNS record.

**Comment by BardiaAfshin** [comment]
@quietlyconfident do you have a sample gist that you can share, that encompasses Route53 DNS Alias record creation for ELB's?

**Comment by quietlyconfident** [comment]
No, I don't.  I gave up on trying to do all this myself and now use Elastic
Beanstalk.  Sorry!

**Comment by ethicalmohit** [comment]
This might help. Just hardcode it according to the region.
 
https://docs.aws.amazon.com/general/latest/gr/rande.html#elb_region.
